sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -287,19 +271,25 @@ async def _management_request_async(self, mgmt_msg: Message, op_type: bytes) ->
                     return response
                 if status_code in [401]:
                     raise errors.AuthenticationException(
-                        "Management authentication failed. Status code: {}, Description: {!r}".format(
-                            status_code, description
+                        errors.ErrorCondition.UnauthorizedAccess,
+                        description="Management authentication failed. Status code: {}, Description: {!r}".format(
+                            status_code,
+                            description
                         )
                     )
                 if status_code in [404]:
-                    raise ConnectError(
-                        "Management connection failed. Status code: {}, Description: {!r}".format(
-                            status_code, description
+                    raise errors.AMQPConnectionError(
```

Comment:
Is this raised to customers? Would that be breaking? Or do we handle this internally?